docs/source/tutorial/tutorial-obtain-schema.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "2. Obtain your GraphQL schema"
+---
+
+## Explore your server's schema
+
+This tutorial uses a modified version of the GraphQL server you build as part of [the Apollo full-stack tutorial](https://www.apollographql.com/docs/tutorial/introduction/). You can visit [`https://n1kqy.sse.codesandbox.io/`](https://n1kqy.sse.codesandbox.io/) to start it up and open the GraphiQL tool to explore its schema:
+
+![the GraphiQL query explorer](images/graphiql.png)
+
+The schema defines which GraphQL operations your server can execute. Click **Schema** on the right-hand side to view a list of the possible queries and mutations you can run:
```

Comment:
I'm gonna update this on my end a bit in the follow-up PR - it's true but it's incomplete, since it defines not just the operations, but the types returned by those operations, and the types the fields on those returned types.